instagram/src/components/PostContainer/Post.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import CommentSection from '../CommentSection/CommentSectionContainer';
```

Comment:
I think you've made an error here - you've called it CommentSectionContainer everywhere but here and on line 22.

Reply:
I have just fixed it. Going back and forth within the files ends up with silly mistakes. Thanks for letting me know :)